subiquity/models/subiquity.py

```diff
@@ -215,17 +211,12 @@ def _cloud_init_config(self):
                 config['ssh_authorized_keys'] = self.ssh.authorized_keys
         if self.ssh.install_server:
```

Comment:
it's unfortunate that I don't think the identity / ssh stuff can be handled completely orthogonally here. Oh well.